src/schemas/ExportAddressDataSchema.ts

```
@@ -28,54 +25,51 @@ import {AddressQR} from "../AddressQR";
  * @since 0.3.0
  */
 class ExportAddressDataSchema extends QRCodeDataSchema {
+  constructor() {
+    super();
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)